mode-apis-base/src/alertingModels.ts

```diff
@@ -183,6 +184,11 @@ export interface AlertRecipientEmail extends AlertRecipient {
     readonly email: string;
 }
 
+export interface AlertRecipientEvent extends AlertRecipient {
+    readonly recipientType: AlertRecipientType.EVENT;
+    readonly eventType: string;
```

Comment:
Can this be any string? Or is there a predefined list of possible string the user can choose from?

Reply:
As of now, we don't have a way to list possible event types.
So, it's okay to leave it just an arbitrary string.

In the backend, eventType is validated by the following regex.

https://github.com/moderepo/platform/blob/86f4f1068e050961419a4e3ff8e94047af6f53be/components/pkg/strutil/strutil.go#L22

Reply:
Ok, when I implement the front end, I will make sure to use the same regex